cmake/templates/libsodiumConfig.cmake.in

```diff
@@ -0,0 +1,27 @@
+if(HUNTER_ENABLED)
+    include(hunter_internal_error)
+    include(hunter_status_debug)
+else()
+    function(hunter_internal_error)
+        message(FATAL_ERROR ${ARGV})
+    endfunction()
+
+    function(hunter_status_debug)
+        message(${ARGV})
+    endfunction()
+endif()
+
+get_filename_component(_install_dir "${CMAKE_CURRENT_LIST_DIR}/../../../" ABSOLUTE)
+
+add_library("libsodium::libsodium" STATIC IMPORTED)
+set_target_properties(libsodium::libsodium PROPERTIES
+    INTERFACE_INCLUDE_DIRECTORIES "${_install_dir}/include"
+    @TARGET_INTERFACE_COMPILE_DEFINITIONS@
+)
+
+# Get all libsodiumTargets-*.cmake files
+get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+file(GLOB TARGET_FILES "${_DIR}/libsodiumTargets-*.cmake")
+foreach(file ${TARGET_FILES})
+    include(${file})
+endforeach()
```

Comment:
http://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file